### PR TITLE
Fix/missing association migration

### PR DESCRIPTION
**Pull request by nakashu**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

Associations where missing migrations

### CODE REVIEW

File: app/migrations/Version20160216003932.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        $this->addSql('CREATE TABLE sylius_association_type (id INT AUTO_INCREMENT NOT NULL, code VARCHAR(255) NOT NULL, name VARCHAR(255) NOT NULL, created_at DATETIME NOT NULL, updated_at DATETIME DEFAULT NULL, UNIQUE INDEX UNIQ_6237029277153098 (code), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
+        $this->addSql('ALTER TABLE sylius_product_association ADD CONSTRAINT FK_48E9CDAB4584665A FOREIGN KEY (product_id) REFERENCES sylius_product (id) ON DELETE CASCADE');
+        $this->addSql('ALTER TABLE sylius_product_association ADD CONSTRAINT FK_48E9CDABB1E1C39 FOREIGN KEY (association_type_id) REFERENCES sylius_association_type (id) ON DELETE CASCADE');
+        $this->addSql('ALTER TABLE productassociation_product ADD CONSTRAINT FK_4F0856B59A049479 FOREIGN KEY (productassociation_id) REFERENCES sylius_product_association (id) ON DELETE CASCADE');
```
Comment (pjedrzejewski) [inline app/migrations/Version20160216003932.php]: This table name looks weird. :/ /cc @lchrusciel

Comment (lchrusciel) [inline app/migrations/Version20160216003932.php]: On it.

Comment (nakashu) [inline app/migrations/Version20160216003932.php]: yeah tried to fixing it before generating but could not find where the name is building up

**Comment by lchrusciel** [comment]
It can be closed in favor of #4182

**Comment by nakashu** [comment]
thanks @lchrusciel

**Comment by pjedrzejewski** [comment]
@nakashu Thanks for reporting! :)

**Pull request closed without merging**